rskj-core/src/main/java/co/rsk/net/SnapshotProcessor.java

```diff
@@ -113,7 +117,11 @@ public SnapshotProcessor(Blockchain blockchain,
         this.syncing = false;
     }
 
-    public void startSyncing(PeersInformation peers, SnapSyncState snapSyncState) {
+    private static byte[] getBytes(byte[] result) {
```

Comment:
why moving this private static to top?

Reply:
As mentioned above, that was the formatter. I'll change back all the changes made by it

Reply:
solved!